Blackjack/OutputFormatter.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Text;
+
+namespace Blackjack
+{
+    public static class OutputFormatter
+    {
+        public static string DisplayHand(Hand hand)
+        {
+            StringBuilder stringBuilder = new StringBuilder();
+            var cards = hand.Cards;
+            for (var i = 0; i < hand.Cards.Count; i++)
+            {
+                stringBuilder.Append(cards[i].Rank); // TODO: cast as int; if statement if face card return the value else int + 1 // check out Yatzy CategoryExtensions

```

Comment:
`Ten`, `Jack`, `Queen`, and `King` are all `10` values, which would be messy to work with in an `Enum` directly. As you may have already figured with the "`check out Yatzy CategoryExtensions`", you could consider some kind of [extension class](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/how-to-create-a-new-method-for-an-enumeration) to return a different string per `Rank`.